lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -88,7 +79,8 @@ class ReportUIFeatures {
     this.json = report;
 
     this._setupMediaQueryListeners();
-    this._setupToolsButton();
+    this._dropDown.setupToolsDropDown();
+    this._dropDown.toolsDropDown.addEventListener('click', this.onToolAction);
```

Comment:
i like that you kept `onToolAction` here in this class. makes sense.


lets rename `onToolAction` to `onDropDownMenuClick` and pass the handler into `setup()` (née `setupToolsDropDown()`).

```js
this._dropDown.setup(this.onDropDownMenuClick);
```

then later in `DropDown`..


```js
  setup(menuClickHandler) {
    this.menuEl.addEventListener('click', menuClickHandler);
    // ...
  }
```

Reply:
I am glad you mentioned this, I was thinking the same thing, done!